### PR TITLE
Sdoe locate files

### PR DESCRIPTION
**Pull request by sotorrio1**
Potentially fixes issue #396

### CODE REVIEW

File: foqus_lib/framework/sdoe/df_utils.py [hunk 1]
```diff
@@ -8,7 +9,7 @@ def write(fname, df):
 
 def load(fname):
     # load file as data frame
-    df = pd.read_csv(fname)
+    df = pd.read_csv(os.path.join('c:/', fname))
```
Comment (ksbeattie) [inline foqus_lib/framework/sdoe/df_utils.py]: This looks OS-specific.  What will happen here on a non-Windows platform?

Comment (sotorrio1) [reply foqus_lib/framework/sdoe/df_utils.py]: That was my first thought, but I tested it on Mac and still works...

Comment (jmorgan29) [reply foqus_lib/framework/sdoe/df_utils.py]: I verified that this fixed the SDOE related aspect of Issue #396

**Comment by sotorrio1** [comment]
@jmorgan29 Let me try to find out the best way to fix this and I'll create a new PR. Glad to hear this little hack works...

**Pull request closed without merging**